frontend/src/metabase/dashboard/actions/parameters.js

```diff
@@ -175,8 +175,10 @@ export const setParameterFilteringParameters = createThunkAction(
 export const SET_PARAMETER_VALUE = "metabase/dashboard/SET_PARAMETER_VALUE";
 export const setParameterValue = createThunkAction(
   SET_PARAMETER_VALUE,
-  (parameterId, value) => (dispatch, getState) => {
-    return { id: parameterId, value };
+  (parameterId, value) => (_dispatch, getState) => {
+    const isSettingDraftParameterValues = !getDashboard(getState())
+      .auto_apply_filters;
```

Comment:
When you define a selector for `auto_apply_filters`, make sure to update this line and re-use it.

Reply:
fixed